lib/granite/rspec/satisfy_preconditions.rb

```diff
@@ -94,10 +94,12 @@ def expected_kind_of_messages_error(object, expected_kind_of_messages, message =
   end
 
   def compare(error, expected)
-    if expected.is_a? String
+    if RSpec::Matchers.is_a_matcher?(expected)
+      expected.matches?(error)
+    elsif expected.is_a?(String)
       error == expected
     else
-      error.match? expected
+      error.match?(expected)
```

Comment:
Wondering if RSpec's built-in `values_match?` does the same thing as this method.

Reply:
This is the first thing I've tried, but tests failed. Didn't dig deeper though

Reply:
Looks good 👍